src/screens/Asset/ReceiveModal.js

```diff
@@ -232,3 +154,60 @@ const combinedMapStateToProps = (state: RootReducerState): $Shape<StateProps> =>
 });
 
 export default (connect(combinedMapStateToProps)(ReceiveModal): AbstractComponent<OwnProps>);
+
+const ContentWrapper = styled(SafeAreaView)`
+  padding: 0 ${spacing.layoutSides}px 30px;
+  align-items: center;
+`;
+
+const QRCodeWrapper = styled.View`
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  margin: ${spacing.largePlus}px;
+`;
+
+const WalletAddress = styled(BaseText)`
```

Comment:
Pls modern/Text component instead of BaseText for new work

Reply:
Done.